tests/test_modeling_gpt2.py

```diff
@@ -332,7 +346,11 @@ def prepare_config_and_inputs_for_common(self):
 @require_torch
 class GPT2ModelTest(ModelTesterMixin, unittest.TestCase):
 
-    all_model_classes = (GPT2Model, GPT2LMHeadModel, GPT2DoubleHeadsModel) if is_torch_available() else ()
+    all_model_classes = (
+        (GPT2Model, GPT2LMHeadModel, GPT2DoubleHeadsModel, GPT2ForSequenceClassification)
+        if is_torch_available()
+        else ()
+    )
     all_generative_model_classes = (
         (GPT2LMHeadModel, GPT2DoubleHeadsModel) if is_torch_available() else ()
     )  # TODO (PVP): Add Double HeadsModel when generate() function is changed accordingly
```

Comment:
Unrelated, is this comment still relevant?

Reply:
comment can be removed => `GPT2DobuleHeadsModel` works with generate()